app/models/container_group.rb

```diff
@@ -37,6 +38,17 @@ def ready_condition_status
     ready_condition.try(:status) || 'None'
   end
 
+  def container_states_summary
+    summary = Hash.new(0)
+    containers.each { |container| summary[container.state.to_sym] += 1 }
+    summary
```

Comment:
Ah btw I just noticed that @matthewd suggested something similar earlier. @zakiva was it not working?